bin/add-roles.sh

```diff
@@ -2,6 +2,8 @@
 
 # User used during the CCD import and ccd-role creation
 ./bin/utils/ccd-add-role.sh "caseworker-civil"
+./bin/utils/ccd-add-role.sh "APPLICANT-PROFILE-SPEC"
```

Comment:
These roles should be added to all other repos other ga ccd import will fail in other repos

Reply:
Done